### PR TITLE
Portfolio-site template 3

### PR DESCRIPTION
**Pull request by HatimDev-HE**
In this feature, i made a portfolio page 1 mobile view.

### CODE REVIEW

**Comment by vic778** [comment]
Do not forget to add more descriptions for your description. Update the REAMED is not enough

**Review by chrystalme — COMMENTED** [review]
# Hi @HatimDev-HE , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❗ 

You have submitted a project, that your PR fails at least one linter test. Kindly fix it and submit another PR.
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mre55 — COMMENTED** [review]
**Hi @HatimDev-HE,**

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :negative_squared_cross_mark:

You have submitted a project, that linter is not passing correctly kindly fix that.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -98,21 +115,24 @@ main {
   padding-right: 0.5rem;
   background-image: url("images/Vector.png.png");
   height: 100vh;
-background-size: cover;
-width: 100%;
+  background-size: cover;
+  width: 100%;
 }
+
```
Comment (tresorsawasawa) [inline styles.css:121]: - [ ] Good job :100: so far with your styling in the heading section. So, I noticed that on your page there are some issues you should work on:
   - Kindly add the `background image` from the Figma design and keep the all spacing match correctly at the top, right, bottom, and left.
   - Use the right spacing between **heading text**, **paragraph** and **social icons**. For that, I think you should check **margins** or **paddings.**
   - And at the **bottom**, you should use the right spacing, i.e margin-bottom of the section.
 
|### Figma design | Your design |
| ------------------------- | -------------------|
| figma_s | s_ds |

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @HatimDev-HE 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes  :recycle::recycle: 
- [ ] Kindy improve both your **pull request title** to be more descriptive about your milestone and its description. for the title, I can suggest you use the milestone project title and for its description, you can add the list of the project's requirements as they are all changes you implemented in the project.
- [ ] Kindly improve your README to be more professional by adding some information such as:
  - #### The description of your project.
  - #### Your social links in the author section 
  - #### And other things to improve your documentation.
- [ ] Check other comments below the review

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -53,10 +53,10 @@
...
         <section id="headline">
           <div id="introduction">
-            <h1>
-              <span id="line1"> Hi there. I'm Hatim</span> <br />
-              <span id="colored">I'm a Software Developer</span>
-            </h1>
+            
+              <h1 id="line1"> Hi there. I'm Hatim</h1> <br />
+              <h1 id="colored">I'm a Software Developer</h1>
+
```
[the remark this answers is not in the document]
Comment (HatimDev-HE) [reply index.html:37]: I am done with changes

File: index.html [hunk 3]
```diff
@@ -12,51 +12,29 @@
...
         <!--headline section-->
         <section id="headline">
           <div id="introduction">
-            <h1>
-              <span id="line1"> Hi there. I'm Hatim</span> <br />
-              <span id="colored">I'm a Software Developer</span>
-            </h1>
+            
+              <h1 id="line1"> Hi there. I'm Hatim</h1> <br />
+              <h1 id="colored">I'm a Software Developer</h1>
```
Comment (m05e5) [inline index.html:36]: - [ ] Please as mentioned by the previous reviewer, its recommended HTML semantics and best practices. So you cannot have 2 `<h1>` tag in the same HTML file so either you replace it with
`<h1 id="line1"> Hi there. I'm Hatim <br />
I'm a Software Developer</h1>
`
or you change the other `h1` to `h2`

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @HatimDev-HE , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve
- [ ] _please check the comments below_

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (HatimDev-HE) [reply index.html]: Okay, I have updated that, I think it's good now. Thank you

**Review by elyor-doniyorov — APPROVED** [review]
Hi @HatimDev-HE, 

In my opinion, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**